examples/workdir/env/extravars

```diff
@@ -19,6 +19,10 @@ redeploy_osh_only: false
 #suse_osh_registry_location: "docker.io"
 #suse_osh_image_version: "pike"
 
+# All helm tests defined for each chart are run by default during deployment.
```

Comment:
Instead of providing those defaults in a file, we should use the documentation.

Reply:
Do you mean move the explanation of the var out of this file and into the documentation? Or do you mean pull everything out of this file and simply add documentation to explain that users can add a var called 'run_tests' and set it to false if they wish to disable them?

Reply:
The latter. I think we should simply give, as example, the bare minimum list of required extra variables. All the optional ones should be documented.

Reply:
Done. I pulled everything out of the extravars example file and added a file in source/user documenting how to disable tests or customize the default test timeout value.